app/pages/popup.html

```diff
@@ -102,38 +134,64 @@
           color: #66d9ef;
         }
       }
-  </style>
-</head>
 
-<body>
-  <div class="container">
-    <div class="header">YT Anti Translate 1.17.0 <small>(Manifest V3)</small></div>
-    <div>
-      Status:
-      <div class="status enabled" id="status">Enabled</div>
-    </div>
-    <button id="disable-button">Disable</button>
-    <div class="options">
-      <div class="option">
-        <span>Untranslate audio track</span>
-        <input id="audio-checkbox" type="checkbox" checked />
+      @keyframes fadeInOut {
+        0% { opacity: 0; }
+        20% { opacity: 1; }
+        80% { opacity: 1; }
+        100% { opacity: 0; }
+      }
+      #save-api-key-text.saving {
+        animation: fadeInOut 1.5s ease;
+      }
+    </style>
+  </head>
+
+  <body id="yt-anti-translate">
+    <div class="container">
+      <div class="header">YT Anti Translate 1.18.1.6<small> (Manifest V3)</small></div>
+      <div>
+        Status:
+        <div class="status enabled" id="status">Enabled</div>
       </div>
-      <div class="option">
-        <span>Untranslate description</span>
-        <input id="description-checkbox" type="checkbox" checked />
+      <button id="disable-button">Disable</button>
+      <div class="options">
+        <div class="option">
+          <span>Untranslate audio track</span>
+          <input id="audio-checkbox" type="checkbox" checked />
+        </div>
+        <div class="option">
+          <span>Untranslate description</span>
+          <input id="description-checkbox" type="checkbox" checked />
+        </div>
+        <div class="option">
+          <span>Reload page automatically</span>
+          <input id="reload-checkbox" type="checkbox" checked />
+        </div>
+        <div class="option" id="api-key-container">
+          <div class="small">
+            <p>Additional features require a <b>YouTube Data API Key</b></p>
+            <p><a target="_blank" href="https://github.com/zpix1/yt-anti-translate/blob/main/YOUTUBE_DATA_API_KEY.md">How do I obtain an API Key?</a></p>
+          </div>    
+          <input id="api-key-input" type="password" placeholder="Enter API key" />
+          <button id="save-api-key-button"><span id="save-api-key-text">Save API Key</span></button>
+        </div>
+        <div class="option bordered-group" id="additional-features" style="display: none;">
+          <small>Additional Features</small>
+          <div class="option">
+            <span>Untranslate channel branding</span>
+            <input id="channel-branding-checkbox" type="checkbox" />
+          </div>
+        </div>
       </div>
-      <div class="option">
-        <span>Reload page automatically</span>
-        <input id="reload-checkbox" type="checkbox" checked />
+      <div class="footer">
+        <a target="_blank" href="https://chrome.google.com/webstore/detail/yt-anti-translate/ndpmhjnlfkgfalaieeneneenijondgag">Rate Chrome extension</a> • 
```

Comment:
too much links, must be at most 3 (check if current browser is firefox/chrome). If you still want donations to be displayed, make donation link direct to not to my site, but to github support section in README.

BTW, no donations even after 30k users, so maybe rm that completly